test-suite/consensus-tests/src/lib.rs

```diff
@@ -585,11 +587,13 @@ impl Sandbox {
 
         for _ in 0..expected_set.len() {
             if let Some((real_addr, real_msg)) = self.pop_sent_message() {
-                assert_eq!(
-                    expected_msg,
-                    real_msg.as_raw(),
-                    "Expected to broadcast other message"
-                );
+                if expected_msg != real_msg.as_raw() {
```

Comment:
Why not use `assert_eq!` with a custom message?

Reply:
Because we compare raw message, but display not raw.

Reply:
Just out of curiosity. Is there some benefit of displaying not raw message here ? In below's panics we display raw messages.

Reply:
Yes, there is: raw message contains payload only as a sequence of bytes, and wrapped message provides not serialized data (e.g. `Propose { ... }`) as well. While I was fixing the bug, I had plenty of panics here and believe me -- raw message in log is really useless :(
I think we should avoid displaying raw messages in panics/assertions everywhere, but it's a topic for a different PR. I'll create an issue for that.